manifests/c/CopyTrans/CopyTransHEIC/1.0.1.0/CopyTrans.CopyTransHEIC.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: CopyTrans.CopyTransHEIC
+PackageVersion: 1.0.1.0
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://www.copytrans.net/download-zip?program=CTH

```

```suggestion
  InstallerUrl: https://www.copytrans.net/bin/CopyTransHEICforWindowsv1.009.exe
```
Direct URL to versioned installer.